docs/media-buy/api-reference.md

```diff
@@ -28,11 +28,22 @@ The Advertising Context Protocol (AdCP) Sales Agent specification defines a stan
 
 ## Core Concepts
 
+### Authentication
+All requests must include authentication via one of:
+- **JWT Bearer Token**: `Authorization: Bearer <token>`
+- **API Key**: `X-API-Key: <key>`
+
+Optional tenant identification:
+- **Header**: `X-Tenant-ID: <tenant_id>`
+- **JWT Claim**: `adcp.tenant_id` in token
+- **Subdomain**: `<tenant>.adcp.com`
+
 ### Principal
 A Principal represents an authenticated entity (advertiser, agency, or brand) with:
```

Comment:
could be an ad network or ad tech company too. brand isn't a real category here (that would be part of an advertiser)